{"name": "困人处置列表-待处理", "status": "passed", "attachments": [{"name": "log", "source": "ae97f796-8405-49af-a6d9-65977d5d30ac-attachment.txt", "type": "text/plain"}], "start": 1692236259026, "stop": 1692236259266, "uuid": "af66d080-3045-4cb1-89e0-91895c67f281", "historyId": "268ac27d44efa67b57ad425fd8b2df94", "testCaseId": "7335a6c89350efea2656ff5aefa50509", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_002", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "13776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}